cordapp-cpk2/src/test/resources/with-dependent-cordapp/build.gradle

```diff
@@ -40,13 +40,3 @@ dependencies {
         transitive = false
     }
 }
-
-// Copy the DependencyConstraints XML from ':cordapp' into our build directory.
-def copy = tasks.register('copy', Copy) {
-    into project.layout.buildDirectory
-    from project(':cordapp').tasks.named('cordappDependencyConstraints')
-}
-
-tasks.named('assemble') {
-    dependsOn copy
-}
```

Comment:
This script meant that `WithDependentCordappTest` didn't need to rummage around in the test project directory structure to find something from the `:cordapp` sub-project. If you want the CPK now instead of the XML then you can do this:
```gradle
configurations {
    cordappDependency {
        canBeConsumed = false
    }
}

// Copy the CPK from ':cordapp' into our build directory.
def copy = tasks.register('copy', Copy) {
    into project.layout.buildDirectory
    from configurations.cordappDependency
}

tasks.named('assemble') {
    dependsOn copy
}
```
And also add this to the `dependencies` block, of course:
```gradle
dependencies {
    ...
    cordappDependency project(path: ':cordapp', configuration: 'cordaCPK')
}
```

Reply:
These changes have been applied.